PackageBuilder/MultiPackageListing.cs

```diff
@@ -98,6 +101,49 @@ partial class Build
                 };
                 string savePath = ListPublishDirectory / PackageListingPublishFilename;
                 repoList.Save(savePath);
+
+                var indexReadPath = PackageListingSourcePath.Parent / "Website" / WebPageIndexFilename;
+                var appReadPath = PackageListingSourcePath.Parent / "Website" / WebPageAppFilename;
+                var stylesReadPath = PackageListingSourcePath.Parent / "Website" / WebPageStylesFilename;
```

Comment:
```suggestion
                AbsolutePath WebPageSourcePath = PackageListingSourcePath.Parent / "Website";
                var indexReadPath = WebPageSourcePath / WebPageIndexFilename;
                var appReadPath = WebPageSourcePath / WebPageAppFilename;
                var stylesReadPath = WebPageSourcePath / WebPageStylesFilename;
```
This is a good candidate for introducing a reusable absolute path for readability and maintainability, I think. Caveat - I wrote this directly on GitHub, code not tested.

Reply:
Agreed, will do and test